### PR TITLE
[MIG] [9.0] base_report_to_printer: Quick migration to Odoo 9

### PR DESCRIPTION
**Pull request by morphace**
as well as a German language file...

### CODE REVIEW

**Comment by pedrobaeza** [comment]
There are commits in 8.0 not present here. Please follow this guide: https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-9.0

File: base_report_to_printer/printing_view.xml [hunk 1]
```diff
@@ -100,12 +100,12 @@
...
-            <field name="printing_action"/>
-            <field name="printing_printer_id"/>
-          </group>
-        </page>
+	      <xpath expr="//notebook/page[2]" position="inside">
+		      <group name="printing">
+			      <field name="printing_action"/>
+			      <field name="printing_printer_id"/>
+		      </group>
+	      </xpath>
```
Comment (yvaucher) [inline base_report_to_printer/printing_view.xml:108]: Please don't replace space by tabs

**Comment by yvaucher** [comment]
Please fix those lint errors:

```
************* Module base_report_to_printer.users

base_report_to_printer/users.py:29: [C8104(class-camelcase), res_users] Use `CamelCase` "ResUsers" in class name "res_users". You can use oca-autopep8 of https://github.com/OCA/maintainer-tools to auto fix it.

************* Module base_report_to_printer.printing

base_report_to_printer/printing.py:29: [R8101(openerp-exception-warning), ] Import `Warning` should be renamed as UserError `from openerp.exceptions import Warning as UserError`

************* Module base_report_to_printer.wizard.update_printers

base_report_to_printer/wizard/update_printers.py:25: [R8101(openerp-exception-warning), ] Import `Warning` should be renamed as UserError `from openerp.exceptions import Warning as UserError`
```

**Comment by lasley** [comment]
Closing to be superseded by #52

**Pull request closed without merging**